src/Symfony/Bundle/TwigBundle/Tests/Functional/AttributeExtensionTest.php

```diff
@@ -16,30 +16,30 @@
 use Symfony\Bundle\TwigBundle\TwigBundle;
 use Symfony\Component\Config\Loader\LoaderInterface;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Exception\LogicException;
 use Symfony\Component\Filesystem\Filesystem;
 use Symfony\Component\HttpKernel\Kernel;
 use Twig\Attribute\AsTwigFilter;
 use Twig\Attribute\AsTwigFunction;
 use Twig\Attribute\AsTwigTest;
 use Twig\Environment;
 use Twig\Error\RuntimeError;
+use Twig\Extension\AbstractExtension;
 use Twig\Extension\AttributeExtension;
 
 class AttributeExtensionTest extends TestCase
 {
-    public function testExtensionWithAttributes()
+    /** @beforeClass */
```

Comment:
Please also add the PHPUnit attribute, to be compatible with PHPUnit 11

Reply:
Good idea, attribute added.

I've also seen the pre condition attributes, but none of them can check a composer package version or if a class exists: https://docs.phpunit.de/en/10.5/writing-tests-for-phpunit.html#skipping-tests-using-attributes

Reply:
Looks weird that they don't have a `RequiresClass` attribute (and probably `RequiresInterface`). It would look consistent with the other attributes they have.